index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/port.css">
+  <title>Patrick Portfolio</title>
+</head>
+<body>
+  <!--header-->
+  <div class="container">
+    <header class="header">
+      <nav class="navbar">
+        <a href="#" class="logo">
+          <img src="./images/Portfolio_Logo.png" alt="My Logo"/></a>
+         <a>
+          <img src="./images/Enabled.png" alt="Menu"/>
+        </a>
+      </nav>
+    </header>
+  <!--main section--> 
+    <div class="main_section">
+      <h1 class="h1">I’m Patrick Ndimukulaga Glad to see you!</h1>
+      <p class="p">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+
+      <div>
+        <p class="media_p">LET'S CONNECT</p>
+        <ul class="media_ul">
+          <li class="media_li">
+            <a href="#"><img src="./images/Twitter-icon.svg" alt="Twitter icon"/></a>
+          </li>
+          <li class="media_li">
+            <a href="#"><img src="./images/Linkedin-icon.svg" alt="Linkedin icon"/></a>
+          </li>
+          <li class="media_li">
+            <a href="#"><img src="./images/Medium-icon.svg" alt="Medium icon"/></a>
```

Comment:
- [ ] For semantic purposes, Kindly wrap the headline section in a `<section>` tag instead of a `<div>`. This will improve the SEO of your page.
you can read more on semantic HTML [here](https://www.pluralsight.com/guides/semantic-html)

Reply:
Ooh thanks for the information